src/components/Widget/components/Conversation/components/Sender/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useRef, useEffect, useState } from 'react';
-import { useSelector } from 'react-redux';
-
+import { useSelector, useDispatch } from 'react-redux';
 import { GlobalState } from 'src/store/types';
+import Counter from '../Counter';
+import { setMsgLength } from '../../../../../../store/actions';
```

Comment:
Shouldn't it be enough to write just `src/store/actions`?

Reply:
I tried that, but it wouldn't compile, so I went this route with it instead.